hazelcast/src/main/java/com/hazelcast/concurrent/lock/LockStore.java

```diff
@@ -36,6 +36,8 @@
 
     int getLockCount(Data key);
 
+    int getLockedEntryCount();
```

Comment:
can you please write some basic tests about `lockstore` --> `getLockedEntryCount`